fix(ci): run pytest for the products dir

skoob13 opened the pull request:
## Problem

pytest doesn't run for the `products/**/*` glob in CI.

## Changes

- Fix the CI to run pytest for the products dir.
- Fix tests.

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

N/A

skoob13 committed:
fix: run pytest for the products dir

skoob13 committed:
fix(editor): tests

skoob13 committed:
fix(early-access-features): tests

skoob13 committed:
fix(revenue-analytics): tests

rafaeelaudibert committed:
fix: Make revenue views work with nested dot access

After recent changes to Data Warehouse, revenue views broke. We didnt notice it yesterday because we weren't running tests for the `products/**` glob

rafaeelaudibert reviewed (approved):
Fixed Revenue Analytics

skoob13 committed:
fix: remove breakpoint statement

github-actions[bot] committed:
Update query snapshots

github-actions[bot] committed:
Update query snapshots

rafaeelaudibert committed:
fix: Fix Experiments query

After DW changes + DW setup changes, these are now breaking and weren't previously detected

github-actions[bot] committed:
Update query snapshots

rafaeelaudibert commented:
@kappa90 @skoob13 FYI, merged this, CI now runs for the `product` subfolder
